hadoop-ozone/dist/src/main/smoketest/s3/bucketdelete.robot

```diff
@@ -43,3 +43,21 @@ Delete non-existent bucket
     ${randStr} =   Generate Ozone String
     ${result} =    Execute AWSS3APICli and checkrc    delete-bucket --bucket nosuchbucket-${randStr}    255
                    Should contain                     ${result}                              NoSuchBucket
+
+Delete bucket with incomplete multipart uploads
+    ${bucket} =                Create bucket to be deleted
```

Comment:
> there are still failures in HA-secure acceptance test, maybe I missed something

The test case fails when using a link, not a real bucket.  Links do not contain any keys, not even multipart upload parts.  Thus they can always be deleted (if allowed by ACL).

Please use `Create bucket` to create a real bucket with random name.

Also, please add `[tags]    no-bucket-type` to this test case, since we do not need to repeat it for various bucket types.

Reply:
> The test case fails when using a link, not a real bucket. Links do not contain any keys, not even multipart upload parts. Thus they can always be deleted (if allowed by ACL).

I see, thank you for the explanation.

> Please use Create bucket to create a real bucket with random name.

> Also, please add [tags]    no-bucket-type to this test case, since we do not need to repeat it for various bucket types.

Understood, updated. Thanks for the help.